### PR TITLE
Add example about using CamelSource

### PR DESCRIPTION
**Pull request by nicolaferraro**
This adds some examples of CamelSources with instructions on how to run them.

### CODE REVIEW

**Comment by nicolaferraro** [comment]
I'm not able to tell if this PR broke CI or it was broken. How can I get details of the errors? Logs don't help.

**Comment by samodell** [comment]
There was a bug (https://github.com/knative/test-infra/issues/638) that is now fixed. Tests are passing now; I'll take a look at this.

**Comment by nicolaferraro** [comment]
Ok, thanks @samodell.

File: docs/eventing/samples/apache-camel-source/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+These samples show how to configure a Camel source. It is a event source that
+can leverage one of the [250+ Apache Camel components](https://github.com/apache/camel/tree/master/components)
+for generating events.
+
+## Prerequisites
+
+1. Setup [Knative Serving](../../../serving).
```
Comment (samodell) [inline docs/eventing/samples/apache-camel-source/README.md]: Suggestion: link to the main install page instead and say "Install Knative Serving and Eventing" as one step.

File: docs/eventing/samples/apache-camel-source/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Now, you can **send messages to your bot** with any Telegram client.
+
+Check again the logs on the message dumper:
+
+```shell
+kubectl logs --selector serving.knative.dev/service=camel-message-dumper -c user-container
+```
+
+Each message you'll send to the bot will be printed by the message dumper as cloudevent.
```
Comment (samodell) [inline docs/eventing/samples/apache-camel-source/README.md]:
```suggestion
Each message you send to the bot will be printed by the message dumper as a cloudevent.
```

File: docs/eventing/samples/apache-camel-source/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+Install the [`source_telegram.yaml`](source_telegram.yaml) resource:
+
+```shell
+kubectl apply -f source_telegram.yaml
+```
+
+Now, you can **send messages to your bot** with any Telegram client.
+
+Check again the logs on the message dumper:
```
Comment (samodell) [inline docs/eventing/samples/apache-camel-source/README.md]: Message dumper was removed and replaced with event display in v0.5; we should update accordingly.

Background: https://github.com/knative/docs/issues/1110

**Comment by nicolaferraro** [comment]
Thanks for the review @samodell. Doc should be updated now.

**Pull request merged**